### PR TITLE
fixes #40

### LINKED ISSUE #40: timestamp parsing fails
``` r
library(riem)
head(riem_measures(
  station = "VOHY",
  date_start = "2014-03-01",
  date_end = "2014-04-05"
))
#> Warning: All formats failed to parse. No formats found.
#> # A tibble: 6 × 32
#>   station valid    lon   lat  tmpf  dwpf  relh  drct  sknt  p01i
#>   <chr>   <dttm> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl> <dbl>
#> 1 VOHY    NA      78.5  17.5  69.8  60.8  73.1   130     4     0
#> 2 VOHY    NA      78.5  17.5  69.8  60.8  73.1   190     4     0
#> 3 VOHY    NA      78.5  17.5  77    62.6  61.1   120     6     0
#> 4 VOHY    NA      78.5  17.5  82.4  64.4  54.6   180     5     0
#> 5 VOHY    NA      78.5  17.5  84.2  64.4  51.5   120     6     0
#> 6 VOHY    NA      78.5  17.5  87.8  64.4  45.9   210     5     0
#> # … with 22 more variables: alti <dbl>, mslp <lgl>, vsby <dbl>, gust <lgl>,
#> #   skyc1 <chr>, skyc2 <chr>, skyc3 <chr>, skyc4 <chr>, skyl1 <dbl>,
#> #   skyl2 <dbl>, skyl3 <dbl>, skyl4 <dbl>, wxcodes <chr>,
#> #   ice_accretion_1hr <lgl>, ice_accretion_3hr <lgl>, ice_accretion_6hr <lgl>,
#> #   peak_wind_gust <lgl>, peak_wind_drct <lgl>, peak_wind_time <lgl>,
#> #   feel <dbl>, metar <chr>, snowdepth <lgl>
```
Changed [this line](https://github.com/ropensci/riem/blob/a4501ec8b336e102d639f1559bbe3a51f99acb26/R/measures.R#L108) to :
```r
  result$valid <- lubridate::as_datetime(result$valid,format="%F %H:%M")
```
to fix

### PR DESCRIPTION
**Pull request by BenoitFayolle**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -22,7 +22,7 @@ BugReports: https://github.com/ropensci/riem/issues
 Imports:
     httr2,
     jsonlite (>= 0.9.19),
-    lubridate (>= 1.5.6),
+    lubridate (>= 1.9.0.9000),
```
Comment (maelle) [inline DESCRIPTION:25]: I now remember we need to add a field called Remotes recording where that version comes from.

```
Remotes:
    tidyverse/lubridate
```

If you use usethis simply run `usethis::use_dev_package("lubridate")`

File: R/measures.R [hunk 2]
```diff
@@ -105,7 +105,7 @@ riem_measures <- function(station = "VOHY",
     return(NULL)
   }
 
-  result$valid <- lubridate::ymd_hm(result$valid)
+  result$valid <- lubridate::as_datetime(result$valid,format="%F %H:%M")
```
Comment (maelle) [inline R/measures.R:108]: do you think we could remove this change since it's a lubridate bug fixed in its dev version?

**Review by maelle — CHANGES_REQUESTED** [review]
Thank you! Two comments

**Comment by BenoitFayolle** [comment]
I'll close and create a new PR because that revert won't look good

**Pull request closed without merging**